recipe/meta.yaml

```diff
@@ -75,12 +75,14 @@ outputs:
       imports:
         - {{ name }}
       requires:
-        - nbsmoke
+        - nbval
```

Comment:
`nbval` is not available on defaults channel yet. So we should skip corresponding tests or add this package

Reply:
Oh good to know! I'll adapt the recipe not to use it.

Reply:
Removed this test, our other packages don't have any nbval/nbsmoke tests.